src/main/java/com/libertymutualgroup/herman/aws/ecs/EcsPush.java

```diff
@@ -710,6 +710,13 @@ private void brokerServicesPrePush(EcsPushDefinition definition, EcsDefaultEnvIn
         brokerDynamoDB(definition);
     }
 
+    private void encryptEnvironmentVariables(EcsPushDefinition definition, String applicationKeyId) {
+        EnvironmentEncryptor encryptor = new EnvironmentEncryptor(kmsClient);
```

Comment:
This is the key logic here -- encrypts variables that have "encrypt: true" on them

Reply:
@c1phr and @jelazos7, what is the need for this? If we are encrypting properties for the app wouldn't we just add {cipher} + encrypted value to the app property files?